editor/export/editor_export_platform.h

```diff
@@ -53,7 +59,7 @@ class EditorExportPlatform : public RefCounted {
 	static void _bind_methods();
 
 public:
-	typedef Error (*EditorExportSaveFunction)(void *p_userdata, const String &p_path, const Vector<uint8_t> &p_data, int p_file, int p_total, const Vector<String> &p_enc_in_filters, const Vector<String> &p_enc_ex_filters, const Vector<uint8_t> &p_key);
+	typedef Error (*EditorExportSaveFunction)(void *p_userdata, const String &p_source_path, const String &p_path, const Vector<uint8_t> &p_data, int p_file, int p_total, const Vector<String> &p_enc_in_filters, const Vector<String> &p_enc_ex_filters, const Vector<String> &p_sign_in_filters, const Vector<String> &p_sign_ex_filters, const Vector<uint8_t> &p_key, bool p_is_signed);
```

Comment:
Likewise for this, I'm thinking a public struct might be something each export plugin can reuse for this.